pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1950,12 +1951,36 @@ func ParseResults(result *resource.ResourceSearchResponse, offset int64) (*v0alp
 		Hits:      make([]v0alpha1.DashboardHit, len(result.Results.Rows)),
 	}
 
+	excludedFields := []string{
+		resource.SEARCH_FIELD_EXPLAIN,
+		resource.SEARCH_FIELD_SCORE,
+		resource.SEARCH_FIELD_TITLE,
+		resource.SEARCH_FIELD_FOLDER,
+		resource.SEARCH_FIELD_TAGS,
+	}
```

Comment:
you can declare this as a var outside of this function.  then it doesn't have to be declared every time you call parseResults

Reply:
while at it, a map would be better